RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 3.8
 -----
+- Dashboard stats: any negative revenue (from refunds for example) for a time period should be shown now.
```

Comment:
I'd be inclined to substitute `should` with `will` :D

Reply:
good idea! updated here: https://github.com/woocommerce/woocommerce-ios/pull/1874/commits/118fcf571f3d6b8c06871224b7b0f54eee1e9b6a